docker/flags.go

```diff
@@ -98,6 +98,7 @@ func init() {
 			{"save", "Save an image to a tar archive"},
 			{"search", "Search for an image on the Docker Hub"},
 			{"start", "Start a stopped container"},
+			{"stats", "Display live container stats based on resource usage"},
```

Comment:
Considering that `stats` accepts multiple containers ID as argument, would the following work for you?

> "Display a live stream of one or more containers resource usage statistics"